### PR TITLE
Update operation_utils.py - Added docstrings

### PR DESCRIPTION
**Pull request by sqali**
Hi @fchollet @chenmoneygithub ,

I have Added docstrings to the required functions. Kindly review and let me know if there are any modifications to be made.

### CODE REVIEW

File: keras_core/ops/operation_utils.py [hunk 1]
```diff
@@ -11,7 +11,23 @@ def compute_pooling_output_shape(
     padding="valid",
     data_format="channels_last",
 ):
-    """Compute the output shape of pooling ops."""
+    """
+    Computes the output shape of pooling operations.
+
+    Args:
+        input_shape (Tuple[int]): The shape of the input.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:18]: Please follow the style of other docstrings

File: keras_core/ops/operation_utils.py [hunk 2]
```diff
@@ -11,7 +11,23 @@ def compute_pooling_output_shape(
     padding="valid",
     data_format="channels_last",
 ):
-    """Compute the output shape of pooling ops."""
+    """
+    Computes the output shape of pooling operations.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:15]: The first line should be above

File: keras_core/ops/operation_utils.py [hunk 3]
```diff
@@ -11,7 +11,23 @@ def compute_pooling_output_shape(
...
+
+    Args:
+        input_shape (Tuple[int]): The shape of the input.
+        pool_size (Tuple[int]): The size of the pooling operation.
+        strides (Tuple[int], optional): The stride of the pooling operation. Defaults to pool_size.
+        padding (str, optional): The padding method, either 'valid' or 'same'. Defaults to 'valid'.
+        data_format (str, optional): The data format, either 'channels_last' or 'channels_first'. Defaults to 'channels_last'.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:22]: Make sure lines are shorter than 80 char

File: keras_core/ops/operation_utils.py [hunk 4]
```diff
@@ -11,7 +11,23 @@ def compute_pooling_output_shape(
...
+
+    Returns:
+        Tuple[int]: The output shape of the pooling operation.
+
+    Example:
+        >>> compute_pooling_output_shape((1, 4, 4, 1), (2, 2))
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:28]: Unident code example

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras_core/ops/operation_utils.py (hunk 1 above)
Comment (sqali) [reply keras_core/ops/operation_utils.py]: Followed the style of other docstrings

File: keras_core/ops/operation_utils.py (hunk 2 above)
Comment (sqali) [reply keras_core/ops/operation_utils.py]: Moved the sentence to the first line

File: keras_core/ops/operation_utils.py (hunk 3 above)
Comment (sqali) [reply keras_core/ops/operation_utils.py]: Reduced all the lines to below 80 chars

File: keras_core/ops/operation_utils.py (hunk 4 above)
Comment (sqali) [reply keras_core/ops/operation_utils.py]: Please check if this has been done correctly.

**Comment by sqali** [comment]
Hi @fchollet ,

I have formatted the docstring as per your input and it has cleared all the checks. Kindly review and let me know if there are any changes to be made.

File: keras_core/ops/operation_utils.py [hunk 5]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
     padding="valid",
     data_format="channels_last",
 ):
-    """Compute the output shape of pooling ops."""
+    """Computes the output shape of pooling operations.
+
+    Args:
+    input_shape: Input shape. Must be a tuple of integers.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:17]: Indent Args section by 4 spaces (see other docstrings)

File: keras_core/ops/operation_utils.py [hunk 6]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
...
+    pool_size: Size of the pooling operation. Must be a tuple of integers.
+    strides: Stride of the pooling operation. Must be a tuple of integers.
+        Defaults to pool_size.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:20]: Code keywords should have backticks

File: keras_core/ops/operation_utils.py [hunk 7]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
...
+    padding: Padding method. Available methods are 'valid' or 'same'.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:21]: These are string keywords, so they should have backticks and double quotes.

File: keras_core/ops/operation_utils.py [hunk 8]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
...
+        Defaults to 'valid'.
+    data_format: String, either 'channels_last' or 'channels_first'.
+        The ordering of the dimensions in the inputs. 'channels_last'
+        corresponds to inputs with shape (batch, height, width, channels)
+        while 'channels_first' corresponds to inputs with shape
+        (batch, channels, height, weight). Defaults to 'channels_last'.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:27]: Tuples are code keywords.

File: keras_core/ops/operation_utils.py [hunk 9]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
...
+
+    Returns:
+        Tuple[int]: The output shape of the pooling operation.
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:30]: Just say "tuple of ints"

File: keras_core/ops/operation_utils.py [hunk 10]
```diff
@@ -11,7 +11,29 @@ def compute_pooling_output_shape(
...
+
+    Examples:
```
Comment (fchollet) [inline keras_core/ops/operation_utils.py:32]: There's only 1 example, so singular

**Comment by sqali** [comment]
Hi @fchollet ,

Thanks for being patient with me. I have updated the changes. Kindly review and advise.

File: keras_core/ops/operation_utils.py [hunk 11]
```diff
@@ -11,7 +11,38 @@ def compute_pooling_output_shape(
...
+    """Computes the output shape of pooling operations.
+
+    Args:
+        input_shape: Input shape. Must be a tuple of integers.
+        pool_size: Size of the pooling operation. Must be a tuple of integers.
+        strides: Stride of the pooling operation. Must be a tuple of integers.
+            Defaults to `pool_size`.
+        padding: Padding method. Available methods are `"valid"` or `"same"`.
+            Defaults to `"valid"`.
+        data_format: String, either `"channels_last`" or `"channels_first`".
```
Comment (AakashKumarNain) [inline keras_core/ops/operation_utils.py:23]: `"channels_lats"` or `"channels_first"`

File: keras_core/ops/operation_utils.py [hunk 12]
```diff
@@ -11,7 +11,38 @@ def compute_pooling_output_shape(
...
+            The ordering of the dimensions in the inputs. `"channels_last`"
+            corresponds to inputs with shape `(batch, height, width, channels)`
+            while `"channels_first`" corresponds to inputs with shape
```
Comment (AakashKumarNain) [inline keras_core/ops/operation_utils.py:26]: Same as above

File: keras_core/ops/operation_utils.py (hunk 11 above)
Comment (sqali) [reply keras_core/ops/operation_utils.py:23]: Hi @AakashKumarNain ,

I can see that data_format has been set as ```channels_last``` and not ```channels_lats```. Can you please confirm if I have to replace ```channels_last``` with ```channels_last``` in the docstring?

Comment (AakashKumarNain) [reply keras_core/ops/operation_utils.py:23]: Sorry it was typo. The main thing to change is that the string should be in quotes. I edited the comment above

**Comment by sqali** [comment]
Hi @AakashKumarNain ,

Thanks for helping me out. Can you please check if the docstring is correct now?

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**